cmd/lncli/commands.go

```diff
@@ -242,6 +242,66 @@ func sendCoins(ctx *cli.Context) error {
 	return nil
 }
 
+var listUnspentCommand = cli.Command{
+	Name:      "listunspent",
+	Category:  "On-chain",
+	Usage:     "List all utxos available for spending",
+	ArgsUsage: "minconfs maxconfs",
```

Comment:
These should be listed under `Flags` as well. Take  a look at other commands where this is done :)

Reply:
Done in 60ae2afc236c6eb3d1095cca9dfed5d9760a23b4, it should now be in line with other commands